subsys/bootloader/fprotect/fprotect_spu.c

```diff
@@ -21,7 +21,7 @@ int fprotect_area(u32_t start, size_t length)
 	for (u32_t i = 0; i < length / SPU_LOCK_REGION_SIZE; i++) {
 		nrf_spu_flashregion_set(NRF_SPU_S,
 				(start / SPU_LOCK_REGION_SIZE) + i,
-				false,
+				true,
```

Comment:
why was it false in the 1st place? Did we experience problems with having it in secure?

Reply:
Where is this function used @oyvindronningstad ?
Could it be that it we don't access B0 resources form anywhere else and that's why we have not seen the error?

Reply:
@sigvartmh I'm not sure, I think it was like that from when the file was created, but I haven't dived deep into the git history

@ioannisg https://github.com/NordicPlayground/fw-nrfconnect-nrf/blob/eef5a368a99e65ee005d3fcf0423df369b5e8b3f/samples/bootloader/src/main.c#L178